core/XH.js

```diff
@@ -327,6 +327,19 @@ class XHClass extends HoistBase {
         return this.acm.themeModel.darkTheme;
     }
 
+    //------------------------
+    // Sizing Mode Support
+    //------------------------
+    /** Set the global sizing mode. */
```

Comment:
We might as well jsdoc these all the way
```suggestion
    /** @param {string} sizingMode - new app-wide sizing mode to apply */
```